tools/tidb-binlog-cluster.md

```diff
@@ -55,6 +55,7 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
 * 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 savepoint，然后导入全量备份，最后启动 Drainer 从 savepoint 开始同步增量数据。
 * Drainer 支持将 Binlog 同步到 MySQL、TiDB、Kafka 或者本地文件。如果需要将 Binlog 同步到其他类型的目的地中，可以设置 Drainer 将 Binlog 同步到 Kafka，再读取 Kafka 中的数据进行自定义处理，参考 [binlog slave client 用户文档](../tools/binlog-slave-client.md)。
 * 如果 TiDB-Binlog 用于增量恢复，可以设置下游为 `pb` 将 binlog 同步到本地文件中，再使用 [Reparo](../tools/reparo.md) 恢复增量数据。
+* 如果下游设置为 `tidb`，将使用 TiDB 的隐藏列 `_tidb_rowid` 同步数据。
```

Comment:
All descriptions about `_tidb_rowid` are removed in #1037 according to [the comment](https://github.com/pingcap/docs/pull/744#issuecomment-443713806). I think perhaps it's still not the time to add this line. Please make sure.

Reply:
deleted